src/schema/rules/sidecars/derivatives/common_derivatives.yaml

```
@@ -41,6 +41,7 @@ MaskDerivatives:
   - dataset.DatasetType == "derivative"
   - suffix == "mask"
   fields:
+    RawSources: required
     Type: recommended
```

@sappelhoff `RawSources` is required for masks in 1.7.0. With BIDS-URI, `RawSources` has generally been deprecated. I suppose this should be:

```suggestion
    Type: recommended
    Sources: recommended
    RawSources: deprecated
```

Do you agree?

---

I accidentally committed this. @sappelhoff let me know if I need to revert

---

agreed, thanks!